README.md

```diff
@@ -35,50 +35,56 @@ If you want to get involved in muCommander or have any question or issue to disc
 If you would like to contribute code, it is required to fork the repository and submit a [pull request](https://help.github.com/en/articles/about-pull-requests).
 Within pull requests, it is possible to review, discuss, and improve the changes until they are ready for production. 
 
-### Code Editing  
+### Code Editing
 After cloning the source code repository from GitHub, you would probably want to import the project to an Integrated Development Environment (IDE) such as Eclipse or IntelliJ.
 
 The code repository of muCommander is comprised of a main project that contains its core functionality and several sub-projects. These projects are Gradle projects. Most of the popular IDEs today allow you to import Gradle projects out-of-the-box or via an IDE plugin. By importing the main project that is located at the root directory of the repository you will get all the required code in the IDE.
 
-### How to Run  
+### How to Run
 The use of the Gradle wrapper significantly simplifies the build from the command line. The following commands can be invoked from the root directory of the repository with no further installation.
 
-You can run the application by typing:  
+You can run the application by typing:
 ```
 ./gradlew run
 ```
 
-It is recommended to run the following command when getting an unclear compilation error:
+It is recommended to run the following command when getting an unclear compilation error, or to be sure running the just modified code:
 ```
 ./gradlew clean run
 ```    
 
-### How to Debug  
-In order to debug muCommander, you first need to configure a port using an environment variable named `DEBUG` (e.g., 5005). Then, you can run a debugger that connects to this port using your favorite IDE (see [an example for doing this with IntelliJ](https://github.com/mucommander/mucommander/wiki/Debug-from-IntelliJ)).
+### How to Debug
+In order to debug muCommander, you first need to configure a port using an environment variable named `DEBUG`.
+Example of running muCommander in debug mode:
+```
+export DEBUG=5005
+./gradlew run
+```
+Then, you can run a debugger that connects to this port using your favorite IDE (see [an example for doing this with IntelliJ](https://github.com/mucommander/mucommander/wiki/Debug-from-IntelliJ)).
 
 
 ### Packaging
-The creation of a DMG file for MAC OS (produced in build/distributions):  
+The creation of a DMG file for macOS (produced in build/distributions):
 ```
-./gradlew dmg
+./gradlew clean dmg -PskipDmgSign
```

Comment:
but the `clean` task is not required for this, why adding it here and below?

Reply:
I have impression that changes may not be properly included in DMG (if bundles were built before the change). And, additionally, `jpackage` complains that .app already exists:
```
build/muCommander.app already exists
        at jdk.jpackage/jdk.jpackage.internal.AppImageBundler.createRoot(AppImageBundler.java:155)
        at jdk.jpackage/jdk.jpackage.internal.AppImageBundler.createAppBundle(AppImageBundler.java:174)
        at jdk.jpackage/jdk.jpackage.internal.AppImageBundler.execute(AppImageBundler.java:91)
        at jdk.jpackage/jdk.jpackage.internal.Arguments.generateBundle(Arguments.java:700)
        at jdk.jpackage/jdk.jpackage.internal.Arguments.processArguments(Arguments.java:565)
        at jdk.jpackage/jdk.jpackage.main.Main.execute(Main.java:92)
        at jdk.jpackage/jdk.jpackage.main.Main.main(Main.java:53)
```

But the real reason was to be sure that recent changes are there.

Reply:
okok